doc/api/esm.md

```diff
@@ -437,10 +437,34 @@ The `--experimental-json-modules` flag is needed for the module
 to work.
 
 ```bash
-node --experimental-modules --entry-type=module index.js # fails
-node --experimental-modules --entry-type=module --experimental-json-modules index.js # works
+node --experimental-modules index.mjs # fails
+node --experimental-modules --experimental-json-modules index.mjs # works
 ```
 
+## Experimental WASM Modules
```

Comment:
```suggestion
## Experimental Wasm Modules
```

Reply:
Great catch thanks. Very good to know for future reference!